test/integration/conftest.py

```diff
@@ -29,6 +34,135 @@ def connection(dsn, user, password, schema):
         dsn=dsn,
         user=user,
         password=password,
+        schema=schema
     )
     yield con
     con.close()
+
+
+@pytest.fixture(scope='session', autouse=True)
+def prepare_database(dsn, user, password):
+    class DockerDataLoader:
+        """Data loader for docker based Exasol DB"""
+
+        def __init__(
+            self,
+            dsn,
+            username,
+            password,
+            container_name,
+            data_directory
+        ):
+            self._logger = logging.getLogger("DockerDataLoader")
+            self._dsn = dsn
+            self._user = username
+            self._password = password
+            self._container = container_name
+            self._data_directory = data_directory
+            self._tmp_dir = f"data-{uuid.uuid4()}"
+
+        @property
+        def data_directory(self):
+            return self._data_directory
+
+        @property
+        def ddl_file(self):
+            return self.data_directory / "import.sql"
+
+        @property
+        def csv_files(self):
+            return self.data_directory.rglob("*.csv")
+
+        def load(self):
+            self._create_dir()
+            self._upload_files()
+            self._import_data()
+
+        def _execute_docker_command(self, command):
```

Comment:
This executes any command, not just docker
```suggestion
        def _execute_command(self, command):
```

Reply:
Thx, I'll change it manually, due to the references.